src/Core/Shop/LogoUploader.php

```diff
@@ -97,7 +98,7 @@ public function update($fieldName, $logoPrefix, array $files = [])
         $files = empty($files) ? $_FILES : $files;
 
         if (isset($files[$fieldName]['tmp_name'], $files[$fieldName]['tmp_name'], $files[$fieldName]['size'])) {
-            if ($error = ImageManager::validateUpload($files[$fieldName], Tools::getMaxUploadSize())) {
+            if ($error = ImageManager::validateUpload($files[$fieldName], Tools::getMaxUploadSize(), ShopLogoSettings::AVAILABLE_LOGO_IMAGE_EXTENSIONS)) {
```

Comment:
Mmmm 🤔 in theory DTOs are used to carry some data around. It feels weird to use it in this LogoUploader class.

Maybe we should extract constant AVAILABLE_LOGO_IMAGE_EXTENSIONS outside of the DTO in a more generic / reusable class ? And DTO would use it too.